integration_tests/CMakeLists.txt

```diff
@@ -153,7 +153,14 @@ RUN(NAME test_math           LABELS cpython llvm)
 RUN(NAME test_numpy_01       LABELS cpython llvm)
 RUN(NAME test_numpy_02       LABELS cpython llvm)
 RUN(NAME test_numpy_03       LABELS cpython llvm)
+<<<<<<< HEAD
```

Comment:
I think there were some conflicts which were unresolved when committed.

Reply:
Got it solved now.